dashboard/app/controllers/learning_goal_evaluations_controller.rb

```diff
@@ -25,6 +24,31 @@ def update
     end
   end
 
+  def get_evaluation
```

Comment:
more of an overall question: what is the difference between `get_evaluation` and `get_or_create_evaluation`?